frontend/src/lib/components/LemonPill/LemonPill.tsx

```diff
@@ -0,0 +1,29 @@
+import { LemonButton } from '../LemonButton/LemonButton'
+import { IconClose } from '../icons'
+
+export type LemonPillProps = {
+    children: React.ReactNode
+    onClick: () => void
+    onDelete: () => void
+}
+
+export function LemonPill({ children, onClick, onDelete }: LemonPillProps): JSX.Element {
```

Comment:
What's the difference between this and a rounded `LemonSnack`? They _seem_ quite similar

Reply:
Yep, exactly. There's not much difference between them. I'll refactor LemonSnack and add a `type`-prop with options `regular` and `pill`.